recipes/perl-unix-processors/meta.yaml

```diff
@@ -0,0 +1,38 @@
+---
+{% set name = "perl-unix-processors" %}
+{% set version = "2.046" %}
+{% set sha256_hash = "3973ebdc44682c9c15c776f66e8be242cb4ff1dd52caf43ff446b74d4dccca06" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/W/WS/WSNYDER/Unix-Processors-{{version}}.tar.gz
+  sha256: {{ sha256_hash }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - perl
+  run:
+    - perl
+
+test:
+  imports:
+    - Unix::Processors
+
+about:
+  home: https://metacpan.org/pod/Unix::Processors
+  license: perl_5
```

Comment:
From https://metacpan.org/pod/Unix::Processors:
>DISTRIBUTION
>
>The latest version is available from CPAN and from http://www.veripool.org/.
>Copyright 1999-2017 by Wilson Snyder. This package is free software; you you can redistribute it and/or modify it under the terms of either the GNU Lesser General Public License Version 3 or the Perl Artistic License Version 2.0.

I'm no Perl expert at all, so I have no idea whether `Perl Artistic License Version 2.0` is equivalent to `perl_5`. Can you confirm that?

Reply:
My understanding was that `perl_5` means "under the same license terms as Perl 5", which is a phrase used in many Perl module distributions. Since Perl 5 is dual-licensed, the code can be used under both the GPL or the Perl Artistic license, cf. https://dev.perl.org/licenses/. Is there support for multi-licensed packages in conda? So could I add multiple licenses as a list under the `about/license` key?